### DISCUSSION TITLE
How To Create In A Parent Directory When group_empty

### DISCUSSION THREAD

**Discussion by przepompownia** [nvim-tree/nvim-tree.lua#1699] [Q&A]
Reproduction:
- set `group_empty = true` 
- Given that `/tmp/x` does not exist yet, run `mkdir -p /tmp/x/y` in shell. 
- in neovim run `:NvimTreeOpen /tmp` and go to `x` node and expand it to see `x/y`
- with default mappings press `a` to create new node.

Currently I get `x` selected as a relative dir. I cannot get selected `y` instead. 

What do you think about selecting relative directory by the cursor position?

I also did not found any function to switch such groupping.

**Comment by alex-courtis** [comment]
See `:help nvim-tree.create_in_closed_folder`.

In the above test, the `a` directory is `/tmp/x/y`.

If you'd like to `a` in `/tmp/x` simply remove the `/y`.

**Comment by przepompownia** [comment]
@alex-courtis 
Indeed, after switching on `create_in_closed_folder` this behavior changes to closer to the described above. Thanks again!

**Reply by alex-courtis** [reply]
> @alex-courtis Indeed, after switching on `create_in_closed_folder` this behavior changes to closer to the described above. Thanks again!

That's great! Please mark the answer.

**Reply by przepompownia** [reply]
Hm, I can't see any widget to mark your comment as the answer (after looking to https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions)...

https://github.com/nvim-tree/nvim-tree.lua/discussions/1703#discussioncomment-4015412 for comparison
![image](https://user-images.githubusercontent.com/11404453/199144001-77e29a21-4549-40da-99df-ef41edcd5d61.png)

**Reply by alex-courtis** [reply]
Ah... this was an idea. Changed the topic and moved to Q&A.
